### PR TITLE
Added cancel_after_time_interval

### LINKED ISSUE #128: Add documentation for timeout based cancellation of search request
[This PR](https://github.com/opensearch-project/OpenSearch/pull/986) introduces the support for timeout based cancellation for search request. It adds request level parameter (`cancel_after_time_interval`) for search and msearch requests and cluster setting `search.cancel_after_time_interval` to control the timeout interval for cancelling the request. The feature is targeted for 1.1 release.

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Fixes #128  

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/rest-api/multi-search.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ All multi-search URL parameters are optional. Some can also be applied per-searc
 Parameter | Type | Description | Supported in metadata line
 :--- | :--- | :---
 allow_no_indices | Boolean | Whether to ignore wildcards that don't match any indices. Default is `true`. | Yes
+cancel_after_time_interval | Time | The time after which the search request will be cancelled. Supported at both parent and child request levels. The order of precedence is:<br> 1. Child-level parameter<br> 2. Parent-level parameter<br> 3. [Cluster setting](../cluster-settings).<br>Default is -1. | Yes
```
Comment (alicejw-aws) [inline _opensearch/rest-api/multi-search.md:62]:
```suggestion
cancel_after_time_interval | Integer | The time in seconds after which the search request will be cancelled. Supported at both parent and child request levels. The order of precedence is:<br> 1. Child-level parameter<br> 2. Parent-level parameter<br> 3. [Cluster setting](../cluster-settings).<br>Default is -1. | Yes
```

**Review by alicejw-aws — APPROVED** [review]
LGTM

File: _opensearch/rest-api/multi-search.md (hunk 1 above)
Comment (Naarcha-AWS) [inline _opensearch/rest-api/multi-search.md:62]: Adjust the cluster settings link to the following format `{{site.url}}{{site.baseurl}}/opensearch/rest-api/cluster-settings`.

**Review by Naarcha-AWS — APPROVED** [review]
Quick comment but otherwise LGTM

File: _opensearch/rest-api/multi-search.md (hunk 1 above)
Comment (kolchfa-aws) [reply _opensearch/rest-api/multi-search.md]: Thank you! Changed links.

**Review by natebower — COMMENTED** [review]
Please see my comments and let me know if you have any questions. Thanks!

**Pull request merged**